### ISSUE TITLE
difference from synthesis using python and c

### ISSUE THREAD

**Issue by Liu-Feng-deeplearning** [MlWoo/LPCNet#5]
Hello， MIWoo，thanks for your improvement of the training Tacotron and LPCnet.

But i find that the synthesis results are not same with python (test_lpcnet.py) and c (lpcnet.c)
Do i make some mistakes?

And i use my training data (8k), and do not change any parameters in the c script. i trained model without taco. 

thank!

**Comment by MlWoo** [comment]
@liufeng900204 you are right. The LPCNet of python need more other params besides the 20-dims features(2 pitch params and 18 barked scaled coefficients) . However, the extra params could be calculated with the 20-dims features.  C code does it by itself but python does not.

**Comment by tuanad121** [comment]
Hi @MlWoo, 
I did analysis/synthesis with LPCNet (model: `lpcnet20h_384_10_G16_80.h5`) to check its efficacy on `CMU_arctic`. 
I synthesized with either `test_lpcnet` C or `test_lpcnet.py`. The sample from `test_lpcnet.py` sounds good, but the sample from `test_lpcnet` sounds bad.  I attached the samples. I wonder if the quality of the output of `test_lpcnet` is as expected or it's me doing something wrong.
[samples.zip](https://github.com/MlWoo/LPCNet/files/4143693/samples.zip)
 Thank you very much.

**Comment by MlWoo** [comment]
@tuanad121  I doubt you did not follow the steps when systhesis the audio with C implementation.
Could you attach the procedures when you systhesis the audio?

**Comment by tuanad121** [comment]
@MlWoo Thanks for your quick responses. Here  is my procedure:
`sox arctic_a0001.wav -r 16000 -c 1 arctic_a0001.s16`
`./dump_data test arctic_a0001.s16 test.f32`
`./test_lpcnet test.f32 test.s16` or `python ./test_lpcnet.py test.f32 test.s16`
`sox -t raw -r 16000 -b 16 -c 1 -L -e signed-integer test.s16 test.wav`
I also compiled `dump_data` and `test_lpcnet` with and without taco=1 but the results are not different. 
[arctic_a0001.wav.zip](https://github.com/MlWoo/LPCNet/files/4146197/arctic_a0001.wav.zip)

**Comment by MlWoo** [comment]
@tuanad121 Did you dump the model pretrained with keras to C files by following the command?
`
./dump_lpcnet.py lpcnet20h_384_10_G16_80.h5
`

**Comment by tuanad121** [comment]
@MlWoo yes I'm. I double checked again and I did `./dump_lpcnet.py` then recompiled everything after `make clean`

**Comment by alokprasad** [comment]
@tuanad121 , can you share your trained model of Tacotron2 for Lpcnet , ?
 Did you trained LPCNET also? 
Is there any difference in sysnthesis using test_lpcnet.py if we use Tacotron2 , as for C code we have extra flag taco=1.
